### DISCUSSION TITLE
Deploy Next.js, Strapi

### DISCUSSION THREAD

**Discussion by kellysciandra** [vercel/next.js#24615] [Help]

**Comment by wwwdepot** [comment]
Where are you hosting Strapi? Make sure it's public accessible on the internet! Use Postman or similar and call a few requests to make sure it's up and running.

**Reply by kellysciandra** [reply]
Thank you so much for answering. Strapi and mongodb backend, next.js front end. Vercel. All settings seem fine but I'm definitely missing something.

**Comment by bobarros** [comment]
This might help you a lot! --> https://betterprogramming.pub/how-to-properly-ask-for-help-in-coding-ad202751aaca
